Description: Remove shebang line from ugreenchat.py
 Packaged ugreenchat.py is placed to /usr/share/doc/uwsgi-common/examples, so
 Lintian warns about relative interpreter path with informational tag
 (I: example-interpreter-not-absolute).
 .
 As user can install only one of uwsgi-python/uwsgi-python3 packages, we have
 no guaranteed path to uWSGI that could be placed into shebang line. So it's
 removed.
Author: Leonid Borisenko <leo.borisenko@gmail.com>
Forwarded: not-needed
Last-Update: 2011-02-01

Index: uwsgi/ugreenchat.py
===================================================================
--- uwsgi.orig/ugreenchat.py	2011-02-01 00:18:18.000000000 +0200
+++ uwsgi/ugreenchat.py	2011-02-01 00:18:48.000000000 +0200
@@ -1,4 +1,3 @@
-#!./uwsgi
 import uwsgi
 
 from bottle import route, default_app, request, response, redirect
